masq/src/commands/set_configuration_command.rs

```diff
@@ -135,16 +128,43 @@ mod tests {
 
     #[test]
     fn command_execution_works_all_fine() {
+        test_command_execution("--start-block", "123456");
+        test_command_execution("--gas-price", "123456");
+        test_command_execution("--min-hops", "6");
+    }
+
+    // TODO: This test only passes when we run through IDE - make it work even without it
```

Comment:
This should be fixed. I may be able to help.

Reply:
Done! 👍